### PR TITLE
✨Ask user for a link to  contributing guide (fixes #168)

### LINKED ISSUE #168: Ask for a link to a contributing guide
CLI should ask for a link to a potential contributing guide.

### PR DESCRIPTION
**Pull request by anku255**
Hi!

This PR fixes #168 

I had to rename `contributingUrl` to `issuesUrl` first and then I used `contributingUrl` for the contributing guide link. 

Please take a look and suggest any changes if required. Thank you 🙂

### CODE REVIEW

File: src/questions/contributing-url.spec.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  it('should return correct question format', () => {
+    const contributingUrl =
+      'https://github.com/kefranabg/readme-md-generator/blob/master/CONTRIBUTING.md'
+    const projectInfos = { contributingUrl }
+
+    const result = askContributingUrl(projectInfos)
+
+    expect(result).toEqual({
+      type: 'input',
+      message: '📄  Contributing guide url (use empty value to skip)',
```
Comment (kefranabg) [inline src/questions/contributing-url.spec.js]:
```suggestion
      message: '🤝  Contributing guide url (use empty value to skip)',
```

File: src/questions/contributing-url.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = projectInfos => ({
+  type: 'input',
+  message: '📄  Contributing guide url (use empty value to skip)',
```
Comment (kefranabg) [inline src/questions/contributing-url.js]:
```suggestion
  message: '🤝  Contributing guide url (use empty value to skip)',
```

**Comment by anku255** [comment]
> As always this is nice job @anku255! I juste made a suggestion about the emoji 😄

Hi @kefranabg !

Thank you for the compliment. I have one silly request. Can you please add me to the all-contributors list for tests and bug reports as well? 😀

**Comment by kefranabg** [comment]
Sure!

@all-contributors please add @anku255 for test,bugs

**Comment by kefranabg** [comment]
Sure!

@all-contributors please add @anku255 for test,bug

**Comment by kefranabg** [comment]
@all-contributors please add @anku255 for test

**Comment by kefranabg** [comment]
@all-contributors please add @anku255 for bug

**Pull request merged**